app/services/api/charts/sector.rb

```diff
@@ -145,7 +145,7 @@ def emissions_data_from_company(company)
 
         {
           name: company.name,
-          data: assessment&.emissions&.transform_keys(&:to_i),
+          data: (assessment&.emissions || {}).transform_keys(&:to_i),
```

Comment:
couple of lines above we have similar line `data: benchmark.emissions.transform_keys(&:to_i)` - maybe unify it?

Reply:
like
```ruby
def emissions_data_as_numbers(emissions)
  return {} unless emissions

  emissions.transform_keys(&:to_i)
end

data: emissions_data_as_numbers(assessment&.emissions)
data: emissions_data_as_numbers(benchmark&.emissions)
```

Reply:
:+1: